testsuite/cli/src/main.rs

```diff
@@ -79,16 +79,14 @@ struct Args {
 }
 
 fn main() {
-    ::libra_logger::Logger::new().init();
-    crash_handler::setup_panic_handler();
     let args = Args::from_args();
 
     let mut logger = ::libra_logger::Logger::new();
-
     if !args.verbose {
         logger.level(::libra_logger::Level::Warn);
     }
     logger.init();
+    crash_handler::setup_panic_handler();
```

Comment:
this needs logger setup, move to after init

Reply:
Hmm I seemed to have missed this when I tried to fix it last week. Thanks for the catch!